src/site/_data/events.js

```diff
@@ -8,26 +8,18 @@ module.exports = async () => {
     const result = await Storyblok.get('cdn/stories', { version, starts_with: 'events' })
     const events = result.data.stories
 
-    return {
-      events: getEvents(events)
-    }
+    return getEvents(events)
```

Comment:
Het hele punt van een `_data` file in Eleventy is dat je globaal toegang hebt tot `events` omdat dit bestand dan events heet. Ik denk dat je hier veel te moeilijk denkt met het asynchroon ophalen in de `_data/site` file.

Je wilt namelijk straks toegang tot drie dingen: 
1. `site.stories` (de pagina's)
2. `site.navigation` (navigatie data)
3. `events` (alle events die we hebben)